UnityProject/Assets/Scripts/Items/Implants/Organs/Bones.cs

```diff
@@ -5,8 +5,12 @@ namespace Items.Implants.Organs
 {
 	public class Bones : BodyPartFunctionality
 	{
-		[SerializeField] private float BloodGeneratedByOneNutriment = 1;
+		[SerializeField] public float BloodGeneratedByOneNutriment = 30;
```

Comment:
make the public getter, why expose the whole field?

Reply:
it's also being set publicly?